spec/dummy/lib/soft_deletable_model_callbacks.rb

```diff
@@ -17,33 +21,41 @@ def self.included(base)
       around_restore :call_around_restore
       after_restore  :call_after_restore
 
+      after_commit :call_after_commit, on: :destroy
+
       def call_before_destroy
-        @before_destroy_called = true
+        @before_destroy_called = next_callback_order
       end
 
       def call_around_destroy
+        @around_destroy_called = next_callback_order
```

Comment:
Oh i'm just want to make sure around is called after `before` and before `after`.